src/Core/Components/Menu/FluentMenu.razor

```diff
@@ -23,6 +23,9 @@
         }
         else
         {
+            if (_contextMenu) {
```

Comment:
Why not to move the `FluentOverlay` from line 8 before `@if (Anchored)`, to have this possibility for all menus? In other words, why to add an Overlay only for `_contextMenu` = `Trigger == MouseButton.Right` ?

Reply:
That lead to errors on the demo page. Some menus don't show or are not clickable then because the Overlay gets in the way

Reply:
Weird. OK. I trust in you :-)